tests/certification/state/sqlserver/go.mod

```diff
@@ -0,0 +1,110 @@
+module github.com/dapr/components-contrib/tests/certification/secretstores/azure/keyvault
+
+go 1.17
+
+require (
+	github.com/dapr/components-contrib v1.4.0-rc2
+	github.com/dapr/components-contrib/tests/certification v1.4.0-rc2
+	github.com/dapr/dapr v1.4.4-0.20211026235832-5e8d7275a35e
+	github.com/dapr/kit v0.0.2-0.20210614175626-b9074b64d233
+	github.com/stretchr/testify v1.7.0
+)
+
+require (
+	contrib.go.opencensus.io/exporter/prometheus v0.2.0 // indirect
+	contrib.go.opencensus.io/exporter/zipkin v0.1.1 // indirect
+	github.com/AdhityaRamadhanus/fasthttpcors v0.0.0-20170121111917-d4c07198763a // indirect
+	github.com/PuerkitoBio/purell v1.1.1 // indirect
+	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
+	github.com/agrea/ptr v0.0.0-20180711073057-77a518d99b7b // indirect
+	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
+	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
+	github.com/andybalholm/brotli v1.0.2 // indirect
+	github.com/antlr/antlr4 v0.0.0-20200503195918-621b933c7a7f // indirect
+	github.com/armon/go-metrics v0.0.0-20190430140413-ec5e00d3c878 // indirect
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/cenkalti/backoff v2.2.1+incompatible // indirect
+	github.com/cenkalti/backoff/v4 v4.1.1 // indirect
+	github.com/cespare/xxhash/v2 v2.1.1 // indirect
+	github.com/dapr/go-sdk v1.2.1-0.20211017032306-de68193d5cd9 // indirect
```

Comment:
`go mod tidy` this, it looks like you have a direct reference on line 40 of the test.

Side note: There's an issue with /ok-to-test where the deduplication task in the existing workflows ignores test changes, so it won't run this.